src/BlackListExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Level51\Autotranslate;
+
+use SilverStripe\Forms\ReadonlyField;
+use SilverStripe\Forms\TextareaField;
+use SilverStripe\ORM\DataExtension;
+
+class BlackListExtension extends DataExtension
+{
+    private static $db = [
+        'BlackListValues' => 'Text'
+    ];
+
+    public function updateCMSFields($fields)
+    {
+        $fields->addFieldsToTab('Root.Translation', [
+            TextareaField::create('BlackListValues', 'BlackList Values')
+                ->setDescription('Enter one value per line, the translatior will ignore these values and keep them as they are.'),
+            ReadonlyField::create('BlackListValuesPreview', 'Preview')
```

Comment:
What is the separate preview field for? Shouldn't the TextareaField be enough?

Reply:
This was just a box to check that everything works - but if you don't think it's necessary, I'll delete it. 🙂